library/declare.mli

```diff
@@ -87,7 +87,9 @@ val exists_name : Id.t -> bool
 
 
 
-(** Global universe names and constraints *)
+(** Global universe contexts, names and constraints *)
 
+val declare_universe_context : polymorphic -> Univ.universe_context_set -> unit
```

Comment:
Why is this not called do_universe_context for uniformity? What does polymorphic mean exactly? Also, it should be a label. We already have too many functions taking unlabeled booleans as arguments, please don't add new ones.